ics/event.py

```diff
@@ -253,6 +256,22 @@ def status(self, value):
             raise ValueError('status must be one of %s' % statuses)
         self._status = value
 
+    @property
+    def classification(self):
+        return self._classification
+
+    @classification.setter
+    def classification(self, value):
+        if value is not None:
+            classifications = ('PUBLIC', 'PRIVATE', 'CONFIDENTIAL', 'iana-token', 'x-name')
```

Comment:
iana-token and x-name are not strings but references to other parts of the spec. They are defined here : https://tools.ietf.org/html/rfc5545#section-3.1
```
iana-token    = 1*(ALPHA / DIGIT / "-")
x-name        = "X-" [vendorid "-"] 1*(ALPHA / DIGIT / "-")
```
I can't remember if we already had to deal with those values before in the project but i'd say we have to support them. They easy and ugly way would be to accept any input, the correct one would be to check the validity of the data.

Reply:
Does any one have valid examples for _x-name_ and _iana-token_? This would make the implementation much more easier.

Reply:
This article can help you reading the syntax of those expressions https://en.wikipedia.org/wiki/Extended_Backus%E2%80%93Naur_form